optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterGenerator.java

```diff
@@ -91,8 +90,9 @@ public RosterGenerator() {
      * For benchmark only
      * @param entityManager never null
      */
-    public RosterGenerator(EntityManager entityManager) {
+    public RosterGenerator(EntityManager entityManager, ShiftFileParser shiftGenerator) {
```

Comment:
What is ShiftFileParser? What does it do.

The variable name called generator but the type is FileParser - that's confusing.

Reply:
ShiftFileParser probably should be renamed to `ShiftGenerator`. It converts `ShiftTemplate` to `Shift`.

Reply:
(FileParser part is from when it parses files in the October 2017 version)

Reply:
Now that ShiftTemplate with have a createShift(start, end) method, is Shiftgenerator/ShiftTemplate still needed?